inscriptions/ISic003533.xml

```diff
@@ -191,6 +196,12 @@
     </facsimile>
     <text>
         <body>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AL">
+                <ab>
+                    <lb/><gap reason="lost" extent="unknown" unit="line" cert="low"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig>sa</orig><g type="interpunct">·</g><expan><abbr>f</abbr><ex cert="low">ilia</ex></expan> p<unclear>i</unclear><supplied reason="lost" cert="low">issima</supplied> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
capitalise the letters inside 'orig'
add space between each word, so after orig and before b, after g and before expan
g should be `<g ref="#interpunct">` and not g type